src/main/webapp/app/markdown-editor/markdown-editor.component.ts

```diff
@@ -239,7 +247,10 @@ export class MarkdownEditorComponent implements AfterViewInit {
             let remainingMarkdownText = this.markdown.slice(0);
 
             /** create string with the identifiers to use for RegEx by deleting the [] of the domainCommandIdentifiers */
-            const commandIdentifiersString = domainCommandIdentifiersToParse.map(tag => tag.replace('[', '').replace(']', '')).join('|');
+            const commandIdentifiersString = domainCommandIdentifiersToParse
+                .map(tag => tag.replace('[', '').replace(']', ''))
+                .map(escapeStringForUseInRegex)
```

Comment:
If we don't do the escape here, domain commands that use special characters lead to an error when creating the Regex down below.
I had this issue for the "(" character of the testCommand.